developer/src/kmcmplib/tests/kmcompxtest.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Tiny frame console app to compile a .kmx from a .kmn
+ */
+
+#include <windows.h>
+#include <stdio.h>
+#include "kmcompx.h"
+#include <iostream>
+#include <vector>
+#include <string>
+#include <sstream>
+using namespace std;
+
+vector < int > error_vec;
+
+int WINAPI msgproc(int line, KMX_DWORD dwMsgCode, char* szText)
+{
+  error_vec.push_back(dwMsgCode);
+  printf("line %d  error %x  %s\n", line, (unsigned int)dwMsgCode, szText);
+	return 1;
+}
+
+int main(int argc, char *argv[])
+{
+  PKMX_STR kmn_file = argv[1];
+  PKMX_STR kmx_file = argv[2];
+
+	if(argc < 3) {
+		puts("Usage: kmcompxtest source.kmn compiled.kmx");
+		return 1;
+	}
+
+	for(char *p = kmn_file; *p; p++) {
+		if(*p == '/') *p = '\\';
```

Comment:
This will need to be adjusted for Linux. Although I'd suggest to work with '/' in this file and replace `\\` with `/`. Windows API methods accept `/` AFAIK and then we don't need to distinguish between forward and backward slash in the rest of this file. But this can be done in a separate PR.